### DISCUSSION TITLE
Winter vegetation

### DISCUSSION THREAD

**Discussion by SimoneBeltramino** [UMEP-dev/UMEP#412] [Q&A]
To simulate a winter scenario how is it recommended to represent the vegetation? 
Can the software take into account, and if so how, the fact that the vegetation in winter has a lower leaf area density (LAD)? 

Thank you, 
Simone

**Comment by biglimp** [comment]
If you are referring to radiation modelling or shading effects, we usually set the transmissivity to 50%

**Reply by SimoneBeltramino** [reply]
I am referring to the calculation of Tmrt. 
So if I understand correctly you consider 50% in winter instead of 3% in summer also for the calculation of SVF?! 
Is the 50% assumption referable to any literature reference?

**Reply by biglimp** [reply]
Correct. Have a look in [Konarska et al](https://link.springer.com/article/10.1007/s00704-013-1000-3).

**Reply by SimoneBeltramino** [reply]
Thank yo so much!
